department-of-veterans-affairs/caseflow
# Adding spreadsheet templates to repo

Connects #5778 

### Testing Plan
1. Go to http://localhost:3000/JudgeAssignmentTemplate.xlsx and confirm the spreadsheet downloads correctly
1. Go to http://localhost:3000/ROAssignmentTemplate.xlsx and confirm the spreadsheet downloads correctly
